### PR TITLE
23: Feature/authors block

### PR DESCRIPTION
**Pull request by aleesteele**
#23 Authors block
- At bottom of post, added title + appropriate styles
- Should I decrease font weight further to match Tucker's design?

### CODE REVIEW

**Comment by aleesteele** [comment]
Fixed!
- Changed `h2-thin` to `h2`. It was set as `h2-thin` in Zeplin I believe, and with the lighter font weight, doesn't seem to match her designs exactly. My question was about whether I should set a lighter font weight than the previous version.
- Removed the Span around the author's name
- Adding more padding-bottom beneath "About the Authors" and `author_bio`

**Comment by jnschrag** [comment]
@aleesteele 
>Changed h2-thin to h2. It was set as h2-thin in Zeplin I believe, and with the lighter font weight, doesn't seem to match her designs exactly. My question was about whether I should set a lighter font weight than the current version, but that's extraneous now as it is now set to h2 :)

Sorry, I wasn't clear. I meant that the element needed to be an h2, with the styles applied being h2--thin. The issue is that you had it as a span tag instead of an h2 element.

**Comment by aleesteele** [comment]
@jnschrag fixed!

File: assets/_sass/components/_authors-block.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*===============================
+=         Authors Block         =
+===============================*/
+
+.authors-block {
+  padding-bottom: 1rem;
+
+  &__title {
+    @extend %h2;
```
Comment (jnschrag) [inline assets/_sass/components/_authors-block.scss]: This should be `%h2--thin`.

**Comment by leezaweb** [comment]
Great work! You don't need line 19, since this property is applied globally. You could use a block element instead of span and yo wouldn't need line 28 of authors-block.scss. Similarly, I'm not sure why line 17's there (inline-block).

**Comment by aleesteele** [comment]
Thank you, @leezaweb! Great catch, I removed the `inline-block` and the transition styles.

The reason why I originally went about styling this with <span> and not a block-level element is because during previous code reviews, I was encouraged not to over-use block elements (basically more divs that necessary). I'm actually not sure what is best practice, @jnschrag?

**Pull request merged**